### DISCUSSION TITLE
Customizing Book Output - no support for LaTeX output?

### DISCUSSION THREAD

**Discussion by anna57394** [quarto-dev/quarto-cli#5049] [Q&A] [answered]
Hi, 

I would like to make some changes to the pdf output of my book project. In the [respective quarto documention](https://quarto.org/docs/books/book-output.html#latex-output), it's recommended to do so by customizing the LaTeX output. However, following the instructions given there, the `_book` directory of my project is wiped clean and I get the following warning. 

`PS C:\Users\...\Book> quarto render --to latex
WARNING: The latex format is not supported by book projects`

Did I get something wrong? Is there any alternative way to render LaTeX output from my book project?

**Comment by rgaiacs** [comment, marked as answer]
Hi @annameiermueller,

What version of quarto are you using? Things works as expected for me with Quarto 1.3.296.

`quarto create-project --type book` produces files to start with.

`quarto render --to latex` render the project to LaTeX. The `.tex` file is available as `_book/book-latex/5049.tex`. `pdflatex _book/book-latex/5049.tex` compiles without error.

Note that `quarto render --to html` will wipe clean the `_book` folder. This is default Quarto behaviour. The documentation has the following instruction to avoid the wipe clean.

> At this point you should probably make a copy or git branch of the _book directory to perform your final LaTeX modifications within (since the modifications you make to LaTeX will not be preserved in your markdown source, and will therefore be overwritten the next time you render).

# Environment

`quarto check` outputs

```
[✓] Checking versions of quarto binary dependencies...
      Pandoc version 3.1.1: OK
      Dart Sass version 1.55.0: OK
[✓] Checking versions of quarto dependencies......OK
[✓] Checking Quarto installation......OK
      Version: 1.3.296
      Path: /opt/quarto/bin

[✓] Checking basic markdown render....OK

[✓] Checking Python 3 installation....OK
      Version: 3.10.6
      Path: /usr/bin/python3
      Jupyter: 5.2.0
      Kernels: python3

(-) Checking Jupyter engine render....Traceback (most recent call last):
  File "/opt/quarto/share/jupyter/jupyter.py", line 21, in <module>
    from notebook import notebook_execute, RestartKernel
  File "/opt/quarto/share/jupyter/notebook.py", line 18, in <module>
    from nbclient import NotebookClient
ModuleNotFoundError: No module named 'nbclient'
[✓] Checking Jupyter engine render....OK
```

**Reply by dragonstyle** [reply]
LaTeX book support is new in Quarto 1.3, FYI.

**Reply by anna57394** [reply]
Problem solved. I did use the wrong version of quarto. Thank you so much!
